src/apps/weblib/interface.js

```diff
@@ -2,131 +2,141 @@ mergeInto(LibraryManager.library, {
   openUrl: function (url) {
     window.open(UTF8ToString(url), '_blank')
   },
-  setCursor: function (cName) {
-    Module.render.canvas().style.cursor = UTF8ToString(cName)
+  setCursor: function (target, cName) {
+    Module.renders[target].canvas().style.cursor = UTF8ToString(cName)
   },
-  canvas_frameBegin: function () {
-    Module.render.frameBegin()
+  callLater: function (func, obj, later) {
```

Comment:
What is the option where we use such a function?
Surely it is a good solution to fix the max time at the code level?
What is the problem if the value is set too high? If it's not a problem, then only a default time value is needed.

Reply:
At C++ code there is a event loop which was a ["busy event loop"](https://github.com/vizzuhq/vizzu-lib/pull/339/files/9b38b0facd5db6b535b4c638ed97eaf7e39252b8#diff-743f718a07efe57b55e20f794a9ccd982d0feeb7837e17954af66b266d62f4a3L400-L402) until now. 
This function arguments are C objects (c function pointer and the attached object for the callback), and the third represent the timeout when needed to call this c function. It [used in interface.cpp:443](https://github.com/vizzuhq/vizzu-lib/pull/339/files/9b38b0facd5db6b535b4c638ed97eaf7e39252b8#diff-1f7d8ec66b278043d960c437f1da31d968b015ff856cce662b90e1a22abc26d6R443), which is only [called at tooltip](https://github.com/vizzuhq/vizzu-lib/pull/339/files/9b38b0facd5db6b535b4c638ed97eaf7e39252b8#diff-9b6aa0ccd686a5b09e16657752159628089ddf88953437f009014cbb44e16fe4R207-R208) with default current time (now --> `0`) argument.

We fix the minimum time of the callback, not the maximum (`Math.max(20, x)` -> `x<20` => `20`, `x>20` => `x`). This will result that it will not spam the js thread.


When tooltip separated from the C++-code and transformed to a js-plugin, the whole callback event handling can be removed. So this is only a temporary improvement before deletion.

Reply:
Egy helyen használjuk a tooltiphez a markerek trackelésénél. Igazából a paraméter nem is kellene, mert settimeout(..., 0)-val equivalens ahogy használjuk

Reply:
Köszi  az infót.

Reply:
I'll remove the Math.max

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/339/commits/7644d6eff8f1ac6b991064e88389ad5642222198